examples/imitation_learning_openbot/run_trained_policy.py

```diff
@@ -79,20 +80,6 @@
     assert os.path.exists(args.episodes_file)
     df = pd.read_csv(args.episodes_file)
 
-    # do some config modifications based on the rendering mode
-    if args.rendering_mode == "baked":
-        rendering_mode_map_str = "_bake"
-        config.defrost()
-        config.SIMULATION_CONTROLLER.CAMERA_SENSOR.FINAL_COLOR_INDIRECT_LIGHTING_INTENSITY = 1.0
-        config.freeze()
-    elif args.rendering_mode == "raytracing":
-        rendering_mode_map_str = "_rtx"
-        config.defrost()
-        config.SIMULATION_CONTROLLER.CAMERA_SENSOR.FINAL_COLOR_INDIRECT_LIGHTING_INTENSITY = 0.0
-        config.freeze()
-    else:
-        assert False
-
     # clean the episode data folder 
     if not args.benchmark:
         shutil.rmtree(args.eval_dir, ignore_errors=True) # remove the previous dataset to prevent data corruption
```

Comment:
Make log statement consistent with `generate_dataset.py`

Reply:
I think now the overall code structure is as similar as possible to the code structure of `generate_dataset.py`